### DISCUSSION TITLE
سوال در مورد حالت restriction یوتیوب

### DISCUSSION THREAD

**Discussion by majd21** [GFW-knocker/gfw_resist_HTTPS_proxy#22] [Q&A]
سلام یه سوالی داشتم
من وقتی با این ابزارتون به یوتیوب وصل میشم حالت restriction بالاجبار توسط ادمین شبکه فعال شده و نمیشه غیر فعالش کرد
میخواستم ببینم ایده ای ندارید که چطور میشه این حالت برداشت؟

**Comment by GFW-knocker** [comment]
چرا مشکل از dns های دستکاری شده توسط ایرانه
که از forcesafesearch.google.com بجای google.com استفاده میکنه
سعی میکنیم با doh حلش کنیم

@majd21 
Map google domains to forcesafesearch.google.com
https://support.google.com/websearch/answer/186669?sjid=11472786603646282090-EU
